packages/orbit-components/src/Textarea/types.d.ts

```diff
@@ -9,7 +9,6 @@ type Resize = "vertical" | "none";
 
 export interface Props extends Common.Globals, Common.SpaceAfter, Common.DataAttrs {
   readonly ref?: React.ForwardedRef<HTMLTextAreaElement>;
-  readonly size?: Common.InputSize;
```

Comment:
Good! Then it should be removed also from "docs/src/__examples__/Textarea/DEFAULT.tsx", from README.md and from the "index.js.flow" file 👍🏻

Reply:
🤦 I always forget those, thanks!!

Reply:
I think "feat", at least we mostly awalys put it under feat, from what I see in the changelog

Reply:
Done!